include/WCSimPrimaryGeneratorAction.hh

```diff
@@ -302,6 +315,25 @@ private:
     }
   }
 
+  // Diego Costas for AmBeGen
+  /*
+  inline void OpenAmBeFile(G4String fileName)
+  {
+    if ( inputAmBeFile.is_open() )
+      inputAmBeFile.close();
+
+    ambeFileName = fileName;
+    inputAmBeFile.open(ambeFileName, std::fstream::in);
+
+    if ( !inputAmBeFile.is_open() ) {
+      G4cout << "AmBe data file " << ambeFileName << " not found" << G4endl;
+      exit(-1);
+    }
+  }
+  */
```

Comment:
remove the commented block if you don't need it

Reply:
👍